src/SignalR/common/Protocols.MessagePack/src/Protocol/MessagePackHubProtocol.cs

```diff
@@ -543,6 +556,8 @@ private void WriteCloseMessage(CloseMessage message, Stream packer)
             {
                 MessagePackBinary.WriteString(packer, message.Error);
             }
+
+            MessagePackBinary.WriteBoolean(packer, message.AllowAutomaticReconnect);
```

Comment:
super nit: we could omit this if it's the default value (false) and save on message size

Reply:
Since this is message pack, and it only adds a byte to a message that should be sent at most once per connection, I'm leaving it as is for simplicity.